### PR TITLE
Update TagBot.yml

### PR DESCRIPTION
**Pull request by CarloLucibello**
Updates the tagbot yaml to [the most recent version](https://github.com/JuliaRegistries/TagBot#setup)

### CODE REVIEW

**Comment by pdeffebach** [comment]
@bkamins I have no idea how to assess this. Does this look good?

File: .github/workflows/TagBot.yml [hunk 1]
```diff
@@ -12,4 +28,6 @@ jobs:
       - uses: JuliaRegistries/TagBot@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          # Edit the following line to reflect the actual name of the GitHub Secret containing your private key
```
Comment (bkamins) [inline .github/workflows/TagBot.yml:31]:
```suggestion
```

File: .github/workflows/TagBot.yml [hunk 2]
```diff
@@ -12,4 +28,6 @@ jobs:
...
           ssh: ${{ secrets.DOCUMENTER_KEY }}
+          # ssh: ${{ secrets.NAME_OF_MY_SSH_PRIVATE_KEY_SECRET }}
```
Comment (bkamins) [inline .github/workflows/TagBot.yml:33]:
```suggestion
```

**Review by bkamins — APPROVED** [review]

**Comment by bkamins** [comment]
Looks good. I just cleaned the comments that were not needed.

**Comment by pdeffebach** [comment]
Thanks!

**Pull request merged**